ionic2/package.json

```diff
@@ -1,9 +1,12 @@
 {
+  "name": "catchemall",
+  "description": "CatchEmAll: An Ionic project",
   "scripts": {
     "start": "ionic serve",
-    "test": "karma start karma.conf.js"
+    "test": "karma start karma.conf.js",
+    "postinstall": "typings install"
```

Comment:
You are using "typings install", but if you want to pull through with the comfortable installation, I think you want to use "node_modules/.bin/typings install", otherwise you're relying on a global installation we are trying to get rid of.